src/namespace/privilegednamespace.js

```diff
@@ -213,4 +219,54 @@ export class PrivilegedNamespace extends CustomNamespace {
         await debate.sendNewQuestion(question);
         callback(question.id);
     };
-}
\ No newline at end of file
+
+    /**
+     * Return an array that contains stats for the debates of an admin in the result of the callback function
+     */
+    getAdminStats = (socket) => async (callback) => {
+        logger.debug(`getAdminStats received from ${socket.id}`);
+
+        if (!(callback instanceof Function)) {
+            logger.debug(`callback is not a function.`);
+            return;
+        }
```

Comment:
I see this block of code appearing quite often. Is it possible to refactor it? Like extracting a method. 

You don't have to apply this change now, I will still approve this PR anyways if there is no other problem with it.

Reply:
The block of code I'm talking about (it wasn't appearing correctly on my view)
```js
if (!(callback instanceof Function)) {
    logger.debug(`callback is not a function.`);
    return;
}
```

Reply:
I will try to change it in the next PR